src/server/routes/controllers/Middlewares/logInUser.js

```diff
@@ -0,0 +1,15 @@
+const strategyUtils = require('../../../passport/strategyUtils')
+
+function logInUser(passport = require('passport')) {
+	return (req, res, next) => {
+		if(req.isAuthenticated())
```

Comment:
It might be more clear if we'd make
`const isAlreadyLoggedIn = req.isAuthenticated()` before if statement, and just use `isAlreadyLoggedIn` in the condition. It will show business logic more clearly.

Reply:
ok